zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/service/ZaakObjectService.kt

```diff
@@ -182,17 +188,43 @@ class ZaakObjectService(
         ).url
     }
 
-    fun deleteObject(objectManagementId: UUID, objectUrl : URI): HttpStatus {
+    fun deleteObject(objectManagementId: UUID, objectUrl: URI): HttpStatus {
         val objectManagementInfo = objectManagementInfoProvider.getObjectManagementInfo(objectManagementId)
 
         val objectenApiPlugin = pluginService.createInstance(
             PluginConfigurationId(
-                objectManagementInfo.objectenApiPluginConfigurationId)
+                objectManagementInfo.objectenApiPluginConfigurationId
+            )
         ) as ObjectenApiPlugin
 
         return objectenApiPlugin.deleteObject(objectUrl)
     }
 
+    fun patchObjectFromManagementId(objectManagementId: UUID, objectId: UUID, jsonNode: JsonNode): URI {
+        val objectManagement = objectManagementInfoProvider.getObjectManagementInfo(objectManagementId)
+
+        val objectenApiPlugin = pluginService.createInstance(
+            PluginConfigurationId.existingId(objectManagement.objectenApiPluginConfigurationId)
+        ) as ObjectenApiPlugin
+
+        val objectTypenApiPlugin = pluginService.createInstance(
+            PluginConfigurationId.existingId(objectManagement.objectenApiPluginConfigurationId)
+        ) as ObjecttypenApiPlugin
+
+        val objectRequest = ObjectRequest(
+            objectTypenApiPlugin.url,
+            ObjectRecord(
+                typeVersion = objectManagement.objecttypeVersion,
+                data = jsonNode,
+                startAt = LocalDate.now()
+            )
+        )
+
+        val objectUrl = URI.create("${objectenApiPlugin.url}objects/$objectId")
```

Comment:
This is not a save way of constructing a url. For example, if the `objectenApiPlugin.url` was configured to be `http://localhost:8010/api/v2`, it would fail.
 
 Please use `UriComponentsBuilder`